waspc/packages/deploy/src/providers/fly/helpers/CommonOps.ts

```diff
@@ -0,0 +1,42 @@
+import { cdToClientBuildDir, cdToServerBuildDir } from "./helpers.js"
+import * as tomlHelpers from "./tomlFileHelpers.js"
```

Comment:
Can do. I was before, but the import list got pretty long. But if it's more common that way can do 👍🏻

Reply:
@sodic are you sure this is bad?
I think this maps to what we can do with Haskell: we can list imports, or we can import them qualified. I actually quite like qualified imports, because when I am reading the code, and I see `tomlHelpers.foo`, I know what what `foo` is about, and I know it is imported and not from my local code. So I don't know, I would actually even encourage this.
Are there are some performance concerns though? Stuff like tree shaking or whatever is the name for that feature where only stuff that you need is loaded? Can we in that case list them but then still scope them under one name?

Reply:
I just changed to explicit imports, all good thanks! 👍🏻